gui/src/components/Onboarding/Carousel.tsx

```diff
@@ -33,6 +33,9 @@ export function OnboardingCarousel({
   const maxSteps = steps.length;
   const step = steps[activeStep];
 
+  const disableNext =
+    activeStep == 3 && !(formData.createTestWallet || formData.addedHDWallet);
```

Comment:
it's not at all clear to me what this variable is suposed to represent

Reply:
well, what about disabling those buttons altogether?
I think we discussed this at some point (though as a separate task), that you shouldn't be able to hop forward/backwards in the flow, as that may mess things up

Reply:
By removing the buttons how are we supposed to advance to the next step/progress on the onboard? Don't know how to proceed with this 🤔